components/Tabs/docs/Examples.md

```diff
@@ -0,0 +1,33 @@
+### Creating a tab bar
```

Comment:
Lower case hyphenated names here and throughout please. This is following the web url convention.

Reply:
done.

Reply:
done.